_posts/myths-facts.md

```diff
@@ -8,49 +8,50 @@ author:
+There are some statements that cities or elected officials have been making that are incorrect regarding DART and their services, funding, or safety. DATA has compiled some of them and debunked every myth.
 
 ---
 ## Myth: DART's sales tax cut won't impact service.
-**Fact**: DART's Chief Communications Officer, Jeamy Molina, has clearly stated that any cut to DART's sales tax could result in reduced services, staff layoffs, and a significant funding loss of nearly **$6 billion** over the next 20 years. Furthermore, the North Central Texas Council of Governments (NCTCOG) does not call for cutting DART sales tax in its Transit 2.0 Request For Proposal (RFP), a study requested by DART's member cities into improving transit for the next 40 years. Instead, it stated:   
-> “Currently there is no evidence that a lower tax rate at DART is possible. If it is possible, what are the impacts to **service, debt obligations and future service commitments**.” 
+**Fact**: DART's Chief Communications Officer, Jeamy Molina, has clearly stated that any cut to DART's sales tax could result in reduced services, staff layoffs, and a significant funding loss of nearly **$6 billion** over the next 20 years. Furthermore, the North Central Texas Council of Governments (NCTCOG) does not call for cutting DART sales tax in its Transit 2.0 Request For Proposal (RFP), a study requested by DART's member cities into improving transit for the next 40 years. Instead, it stated:
+> “Currently there is no evidence that a lower tax rate at DART is possible. If it is possible, what are the impacts to **service, debt obligations and future service commitments**.”
 
-Without careful consideration, these cuts could jeopardize critical projects like the Silver Line or the Tier 2 Bus Network.  
+Without careful consideration, these cuts could jeopardize critical projects like the Silver Line or the Tier 2 Bus Network.
 *Source: [NCTCOG Transit 2.0 RFP](https://www.nctcog.org/getmedia/89832cd2-9626-44c8-bb9b-d868febce6f0/RFP-_Regional-Transit-2-0-FINAL.pdf), [KDFW FOX 4](https://www.fox4news.com/news/farmers-branch-dart-trash-money), [WFAA ABC 8](https://www.wfaa.com/article/news/local/dallas-county/farmers-branch-councilman-dart-trash-comment/287-8e2c934d-5c2c-4c14-99f4-50da22838c74)*
 
 ## Myth: DART is not expanding its services.
-**Fact**: DART's phase 2 (also known as tier 2) bus network is giving all member cities a massive boost in bus service, especially on the highest performing routes. This includes many over-performing suburban bus routes that DART is rewarding with extra frequent service. The phase 2 bus network gives the average DART resident access to 30% more jobs within an hour commute. DART has started implementing their new bus network this year with new bus routes in Irving and Plano, and with frequency boosts on the light rail.  

```

Comment:
Why remove the two spaces after various lines in this file? The two spaces indicate line breaks to markdown.

Reply:
Autoformatting I'm going to reset it after I'm done

Reply:
Yes, the double spaces was required to ensure that the sources are put directly under the image or fact

Reply:
Ok resetted and I've configured vscode to not do that on markdown files so it will never happen again 🙂 

Dumb syntax imo, it should've been // or something like that